static/app/views/alerts/issueRuleEditor/ruleNode.tsx

```diff
@@ -71,7 +71,9 @@ class RuleNode extends React.Component<Props> {
           initialVal = fieldConfig.choices[0][0];
         }
       } else {
-        initialVal = data[name];
+        initialVal = isNaN(parseInt(data[name] as string, 10))
```

Comment:
@leeandher I don't see `name` defined in either IssueAlertRuleAction or IssueAlertRuleCondition types. What is it supposed to be?

Reply:
@NisanthanNanthakumar  It's the name of the dynamic fields that are custom to the integration (e.g. 'workplace' for slack, 'account' or 'service' for pagerduty)